pandas/core/arrays/datetimelike.py

```diff
@@ -80,6 +87,70 @@ def _get_attributes_dict(self):
         """
         return {k: getattr(self, k, None) for k in self._attributes}
 
+    @property
+    def _scalar_type(self):
+        """The scalar associated with this datelike
+
+        * PeriodArray : Period
+        * DatetimeArray : Timestamp
+        * TimedeltaArray : Timedelta
+        """
+        # type: # () -> Union[type, Tuple[type]]
+        raise AbstractMethodError(self)
+
+    def _scalar_from_string(self, value):
```

Comment:
is this a part of the EA interface?

Reply:
Nope. I opened an issue to discuss it https://github.com/pandas-dev/pandas/issues/24261